library/ziti.c

```diff
@@ -239,6 +239,18 @@ void ziti_set_unauthenticated(ziti_context ztx) {
     free_ziti_api_session(ztx->api_session);
     FREE(ztx->api_session);
     ztx->api_session_state = ZitiApiSessionStateUnauthenticated;
+
+    ctrl_clear_api_session(&ztx->controller);
```

Comment:
Clear API Session Token in controller client on 'unauthenticated' and below on 'impossible to authenticate'